apps/condo/domains/user/utils/limits/resetters/AuthGuardResetter.js

```diff
@@ -1,11 +1,61 @@
-const { IPv4_TYPE } = require('@condo/domains/user/constants/identifiers')
+const { EMAIL_TYPE, PHONE_TYPE, UUID_TYPE, IPv4_TYPE } = require('@condo/domains/user/constants/identifiers')
 const { AUTH_COUNTER_LIMIT_TYPE } = require('@condo/domains/user/constants/limits')
 
 const { RedisGuardResetter } = require('./RedisGuardResetter')
 
+
 class AuthGuardResetter extends RedisGuardResetter {
     constructor () {
-        super(AUTH_COUNTER_LIMIT_TYPE, [IPv4_TYPE])
+        super(AUTH_COUNTER_LIMIT_TYPE, [IPv4_TYPE, UUID_TYPE, PHONE_TYPE, EMAIL_TYPE])
+    }
+
+    #getKeys (identifierType, identifier) {
+        const keys = []
+        if (identifierType === 'ip') {
+            keys.push([AUTH_COUNTER_LIMIT_TYPE, 'ip', identifier].join(':'))
```

Comment:
best to consult with infra team on new redis key names before merging this PR

Reply:
As far as I understand, we don't need to change anything